### DISCUSSION TITLE
Support for TSaveDialog

### DISCUSSION THREAD

**Discussion by zeo101** [Embarcadero/DelphiVCL4Python#28] [Ideas]
Hey, 
because of the perfectly working TOpenDialog implementation, I just wanted to ask if there are any plans on adding support for the TSaveDialog?

Kind regards
Julian

**Comment by lmbelo** [comment]
Hello @zeo101,

We are working on new components delivery and we definitly would like to include the TSaveDialog for VCL.
I will let you know when it will be delivered.

Thank you for your interest 👍 

See you,
Lucas

**Reply by zeo101** [reply]
Perfect, thanks for the quick answer.

**Comment by edgebyte-ai** [comment]
Expecting new release.  One month passed.

**Reply by lmbelo** [reply]
@zwm88,
I appologise such delay.
The TSaveDialog has been created and it is now part of the wrapped components, as you can see in the link bellow:
https://github.com/Embarcadero/python4delphi/commit/2e44f4b6dcc11448bfc40ddb8a39bc0c2bcd0399
Unfortunatelly we don't released a new version yet. There are others components still under development.
We are planning a major release with the last requested components.
I will let you know any further update.
